BunqSdk.Tests/Model/Generated/Endpoint/AvatarTest.cs

```diff
@@ -58,8 +58,8 @@ private static string UploadAvatarAndGetUuid(byte[] fileContentByte)
         {
             var customHeaders = new Dictionary<string, string>
             {
-                {ApiClient.HEADER_ATTACHMENT_DESCRIPTION, ATTACHMENT_DECSRIPTION},
-                {ApiClient.HEADER_CONTENT_TYPE, CONTEN_TYPE},
+                {ApiClient.HeaderAttachmentDescription, ATTACHMENT_DECSRIPTION},
+                {ApiClient.HeaderContentType, CONTEN_TYPE},
```

Comment:
`CONTEN_TYPE` is still in caps.

There seems to be a few constants in this file that have not been refactored, please refactor them.